packages/terra-application-name/src/ApplicationMenuName.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+
+import styles from './ApplicationMenuName.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The accessory element to be displayed next to the title.
+   * */
+  accessory: PropTypes.element,
+  /**
+   * The title branding of the application name.
+   * */
+  title: PropTypes.string,
```

Comment:
On this issue, I proposed different ways we can account for overflowing text. (https://github.com/cerner/terra-core/issues/355).

While we don't need to implement these here, I think it will be easier to allow for developers to opt in and out of text truncation by making this a react node as opposed to just a node. In addition, React 16 allows to render a string. Given that, is it possible to change the proptype of the title argument?